packages/cli/src/services/orchestration.handler.service.ts

```diff
@@ -0,0 +1,47 @@
+import Container, { Service } from 'typedi';
+import { RedisService } from './redis.service';
+import type { RedisServicePubSubSubscriber } from './redis/RedisServicePubSubSubscriber';
+import {
+	COMMAND_REDIS_CHANNEL,
+	EVENT_BUS_REDIS_CHANNEL,
+	WORKER_RESPONSE_REDIS_CHANNEL,
+} from './redis/RedisServiceHelper';
+import { handleWorkerResponseMessage } from './orchestration/handleWorkerResponseMessage';
+import { handleCommandMessage } from './orchestration/handleCommandMessage';
+import { MessageEventBus } from '../eventbus/MessageEventBus/MessageEventBus';
+
+@Service()
+export class OrchestrationHandlerService {
```

Comment:
would be great if you can please add tests and coverage for all this new functionality

Reply:
thats all part of the orchestration service tests. it's just refactored and split in two to get rid of the dependency cycles.

Reply:
thanks for explaining.. sorry not enough context to review PR completely but just making sure we are adding tests